core/src/local_cluster.rs

```diff
@@ -23,18 +24,22 @@ pub struct LocalCluster {
     pub entry_point_info: ContactInfo,
     pub ledger_paths: Vec<String>,
     fullnodes: Vec<Fullnode>,
+    replicators: Vec<Replicator>,
+    genesis_ledger_path: String,
+    genesis_block: GenesisBlock,
 }
 
 impl LocalCluster {
     pub fn new(num_nodes: usize, cluster_lamports: u64, lamports_per_node: u64) -> Self {
         let stakes: Vec<_> = (0..num_nodes).map(|_| lamports_per_node).collect();
-        Self::new_with_config(&stakes, cluster_lamports, &FullnodeConfig::default())
+        Self::new_with_config(&stakes, cluster_lamports, &FullnodeConfig::default(), 0)
     }
 
     pub fn new_with_config(
```

Comment:
wdyt about keeping `new_with_config()` as is but then also adding `new_with_config_replicators()`.  The `,0)` hanging off the end of most of the `new_with_config()` calls is a little distracting/confusing to me, and gives an excuse for the next person to add another thing at the end of `new_with_config()`

Reply:
Will do that.